src/app/pages/common/entity/entity-table/entity-table.component.ts

```diff
@@ -560,6 +592,35 @@ export class EntityTableComponent implements OnInit, AfterViewInit, OnDestroy {
       this.paginationPageIndex  = 0;
     }
 
+    if(this.currentRows && this.currentRows.length > 0) {
+      this.isTableEmpty = false;
+    } else {
+      this.isTableEmpty = true;
+      if(this.firstUse) {
+        this.emptyTableConf = {
+          type: EmptyType.first_use,
+          large: true,
+          title: "No "+this.title,
+          message: `It seems you haven't setup any ${this.title} yet.`
```

Comment:
these strings are not translateable, consider splitting up and adding T() to the text

Reply:
Fixed strings